libsrc/atari5200/cputc.s

```diff
@@ -11,13 +11,13 @@
         .export         _cputcxy, _cputc
         .export         plot, cputdirect, putchar
         .import         gotoxy, _mul20
+        .import         conio_color
+        .importzp       screen_width, screen_height
         .importzp       ptr4
-        .import         setcursor
-
-        .constructor    screen_setup, 26
-        .import         screen_setup_20x24
-screen_setup    = screen_setup_20x24
 
+        .import         initconio
+        .constructor    initconio_constructor
```

Comment:
My propsoal was that the "internal" functions have "some" name like e.g. screen_setup and whatever and that the constructor (that goes into the Wiki) has the name "initconio" (not initconio_constructor).

Reply:
you mean like just committed?

Reply:
Yes, exactly :-)